File: Selenium/Project/Activity7.py
from selenium import webdriver
from selenium.webdriver.common.by import By
from webdriver_manager.firefox import GeckoDriverManager

service = webdriver.FirefoxService(GeckoDriverManager().install())

with webdriver.Firefox(service=service) as driver:
    driver.get("https://alchemy.hguy.co/lms")
    driver.find_element(By.XPATH, "//a[text()='All Courses']").click()
    courses = driver.find_elements(By.TAG_NAME, "h3")

    print("The number of courses displayed are : ", len(courses))
    print("The courses names are as below : ")
    for topics in courses:
        print(topics.text, "\n")

    driver.quit()

File: Selenium/Project/Activity7.java
package projects;

import io.github.bonigarcia.wdm.WebDriverManager;
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.firefox.FirefoxDriver;

import java.util.ArrayList;
import java.util.List;

public class Activity7 {
    public static void main(String[] args) {
        System.setProperty(FirefoxDriver.SystemProperty.BROWSER_LOGFILE, "/dev/null");
        WebDriverManager.firefoxdriver().setup();
        WebDriver driver = new FirefoxDriver();

        driver.get("https://alchemy.hguy.co/lms");

        driver.findElement(By.xpath("//a[text()='All Courses']")).click();

        List<WebElement> courses = driver.findElements(By.tagName("h3"));

        System.out.println("The number of courses displayed are : " + courses.size());
        System.out.println("The courses names are as below : ");
        for(WebElement topics : courses){
            System.out.println(topics.getText() + "\n");
        }

        driver.quit();
    }

}
